p_sets/p_set_1.rb

```diff
@@ -51,5 +51,41 @@ def mario
   pyramid_printer calculate_row_length ask_height
 end
 
+#TIMEFORCHANGE
+#assume that your program’s input will be 9.00 or just 9 but, again, not $9 or 900.
+
+COINS = [100,20,10,5,2,1]
```

Comment:
This code appears once above and once here, so I'll just review it here.

Should this be `[100,50,20,10,5,2,1]` for UK? Or `[100,25,10,5,1]` for US?